app/views/people/show.html.erb

```diff
@@ -25,12 +25,29 @@
   <%= tel_to @person.phone_number %>
 </p>
 
+<% person_loc = @person.corresponding_user&.last_known_location %>
+<% if not person_loc.nil? %>
+  <p>
+    <strong>Last known location:</strong>
+    <a href="/route?dest=<%= person_loc %>">
+      <%= person_loc %>
+    </a>
+  </p>
+<% end %>
+
 <p>
   <strong>Rooms:</strong>
-  <ul>
+  <ul class="container">
       <% @person.rooms.each do | room | %>
-        <li>
-          <%= link_to room.name, room_path(room) %>
+        <li class="row">
+          <span class="col">
+            <%= link_to room.name, room_path(room) %>
+          </span>
+          <span class="col">
+            <a class="btn btn-outline-primary btn-sm" href="/route?dest=<%= room.name %>">
```

Comment:
> I couldn't really get the "Directions" buttons to float / justify / align to the right completely, maybe some CSS guru here has an idea? If not, I think it looks okay enough right now

I didn't try it out, but here are two wild guesses :

```suggestion
          <span class="col-auto ms-auto">
            <a class="btn btn-outline-primary btn-sm" href="/route?dest=<%= room.name %>">
```

```suggestion
          <span class="col">
            <a class="btn btn-outline-primary btn-sm float-end" href="/route?dest=<%= room.name %>">
```

Reply:
Huh, I though I tried your first suggestion yesterday :thinking: Well, both work very well, thank you :)

Do you by any chance know where the bullet points went? :D